plugins/jobs/web_client/js/views/JobDetailsWidget.js

```diff
@@ -1,12 +1,26 @@
 girder.views.jobs_JobDetailsWidget = girder.View.extend({
     initialize: function (settings) {
         this.job = settings.job;
-        this.job.on('change', this.render, this);
 
         girder.eventStream.on('g:event.job_status', function (event) {
-            var job = event.data;
-            if (job._id === this.job.id) {
-                this.job.set(job);
+            var info = event.data;
+            if (info._id === this.job.id) {
+                this.job.set(info);
+                this.render();
+            }
+        }, this);
+
+        girder.eventStream.on('g:event.job_log', function (event) {
+            var info = event.data;
+            if (info._id === this.job.id) {
+                var container = this.$('.g-job-log-container');
+                if (info.overwrite) {
+                    this.job.set({log: [info.text]});
+                    container.text(info.text);
```

Comment:
why do you escape the `info.text` for `container.append` below but not this `container.text`?